.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+/vendor
```

Comment:
for me is a mast upload the vendoring

Reply:
don't see much profit from it. With proposed the flow you have to compile frontend after go-get anyway. Running `dep ensure` there doesn't make it worse.
But not committing it allows using the tool with other frontend or whatever another crazy use case when people want to have their own vendoring.
Correct me if I'm wrong but before if your dependency has `vendor` directory you could have a conflict of dependencies.

Still if everybody wants to have `vendor` in repo, it's not _such_ a big deal for me.

Reply:
As I said is standard in go.

Reply:
@campoy you were a go evangelist, right? Can you give me more information about it, please? All I read on the internet, for example, https://golang.github.io/dep/docs/FAQ.html#should-i-commit-my-vendor-directory says it's up to project.

Reply:
This says everything https://github.com/golang/dep/tree/master/vendor

Reply:
That's a great question, actually. So good it's part of the FAQ of dep.

From the [docs](https://golang.github.io/dep/docs/FAQ.html#should-i-commit-my-vendor-directory):

> ## Should I commit my vendor directory?
> 
> It's up to you:
> 
> ### Pros
> 
> It's the only way to get truly reproducible builds, as it guards against upstream renames, deletes and commit history overwrites.
> You don't need an extra dep ensure step to sync vendor/ with Gopkg.lock after most operations, such as go get, cloning, getting latest, merging, etc.
>
> ### Cons
> 
> Your repo will be bigger, potentially a lot bigger, though prune can help minimize this problem.
> PR diffs will include changes for files under vendor/ when Gopkg.lock is modified, however files in vendor/ are hidden by default on Github.

So I'll let @mcuadros decide, since he's the CTO. I do think we should commit vendored dependencies, since this allows for anyone to simply do `go get` of our repo and everything will simply work.

Reply:
the problem is it wouldn't just work.
but no problem. I don't really have a strong opinion on it.
Just wanted some guideline for my projects outside of src-d.

Reply:
As I wrote, almost every single relevant project is committing the vendor. So please commit the vendor.